content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -211,6 +213,8 @@ The following means can be used to troubleshoot the reason why a pod could not b
 - pod status - indicates topology affinity errors
 - system logs - include valuable information for debugging, e.g., about generated hints
 - state file - the dump of internal state of the Memory Manager (includes [Node Map and Memory Maps][2]) 
+- starting from v1.22, the [Device plugin resource API](#device-plugin-resource-api) can be used to retrieve information about the memory reserved for containers

```

Comment:
Nit: not need to capitalize 'device'; wrapping the long line would be desirable.

```suggestion
- starting from v1.22, the [device plugin resource API](#device-plugin-resource-api) can be used
  to retrieve information about the memory reserved for containers

```